### PR TITLE
[LOOP-4853] Date in event history

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-4853

Screenshot 2024-04-19 at 13 55 54

### CODE REVIEW

File: Loop/Managers/Alerts/UserNotificationAlertScheduler.swift [hunk 1]
```diff
@@ -65,7 +65,7 @@ fileprivate extension Alert {
         let userNotificationContent = UNMutableNotificationContent()
         userNotificationContent.title = backgroundContent.title
         userNotificationContent.body = backgroundContent.body
-        userNotificationContent.sound = userNotificationSound(muted: muted)
+        //userNotificationContent.sound = userNotificationSound(muted: muted)
```
Comment (ps2) [inline Loop/Managers/Alerts/UserNotificationAlertScheduler.swift]: Is this intentional? If so, what's it about?

**Review by ps2 — COMMENTED** [review]
I think the section code looks ok. Appears to not handle more than 24 hours, though? Probably fine for now. Just wondering about the alert sound change.

File: Loop/Managers/Alerts/UserNotificationAlertScheduler.swift (hunk 1 above)
Comment (nhamming) [reply Loop/Managers/Alerts/UserNotificationAlertScheduler.swift]: thanks. not intentional

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**